ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2024,6 +2024,7 @@ logging:
     timestamp: < high-resolution | traditional >
     hostname: < fqdn | ipv4 >
     sequence_numbers: < true | false >
+  facility: < syslog facility value >
```

Comment:
I am not sure but some people might get confused, if this is a single value or multiple. So please rephrase this to ```syslog_facility_value```